libmysqlclient building doesn't support ++ in the path

While building qtbase in Fedora 23, I get this error for libmysqlclient:

MXE Log:
```
make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
make[1]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe'
uname -a
Linux KrahlPC.krahl 4.2.6-301.fc23.x86_64 #1 SMP Fri Nov 20 22:22:41 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
git show-branch --list --reflog=1
  [refs/heads/master@{0}] (vor 65 Minuten) (none)
lsb_release -a 2>/dev/null || sw_vers 2>/dev/null || true
LSB Version:	:core-4.1-amd64:core-4.1-ia32:core-4.1-noarch:cxx-4.1-amd64:cxx-4.1-ia32:cxx-4.1-noarch:desktop-4.1-amd64:desktop-4.1-ia32:desktop-4.1-noarch:languages-4.1-amd64:languages-4.1-noarch:printing-4.1-amd64:printing-4.1-noarch
Distributor ID:	Fedora
Description:	Fedora release 23 (Twenty Three)
Release:	23
Codename:	TwentyThree
autoconf --version 2>/dev/null | head -1
autoconf (GNU Autoconf) 2.69
automake --version 2>/dev/null | head -1
automake (GNU automake) 1.15
rm -rf   '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static'
mkdir -p '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static'
cd '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static' &&  tar xzf '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/pkg/mysql-connector-c-6.1.6-src.tar.gz'
cd '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src'
(cd '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src' && patch -p1 -u) < /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/src/libmysqlclient-1-fixes.patch
patching file extra/CMakeLists.txt
patching file libmysql/authentication_win/handshake.h
patching file extra/yassl/src/yassl_int.cpp
patching file include/mysql/psi/mysql_socket.h
patching file libmysql/authentication_win/CMakeLists.txt
patching file libmysql/authentication_win/plugin_client.cc
patching file mysys/my_gethwaddr.c
patching file include/m_string.h
patching file include/my_global.h
patching file include/thr_cond.h
patching file mysys/stacktrace.c
patching file libmysql/CMakeLists.txt
patching file scripts/CMakeLists.txt
patching file include/my_global.h
# native build for tool comp_err
# See https://bugs.mysql.com/bug.php?id=61340
mkdir '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
cd '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native' && cmake '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src'
-- Running cmake version 3.0.2
-- The C compiler identification is GNU 5.1.1
-- The CXX compiler identification is GNU 5.1.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Looking for SHM_HUGETLB
-- Looking for SHM_HUGETLB - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void *
-- Check size of void * - done
-- SIZEOF_VOIDP 8
-- LibMySQL 6.1.6
-- Built from MySQL 5.7.6-m16 sources
-- Packaging as: mysql-connector-c-6.1.6-Linux-x86_64
-- Installing to: /usr/local/mysql
-- Performing Test HAVE_LLVM_LIBCPP
-- Performing Test HAVE_LLVM_LIBCPP - Failed
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Looking for floor
-- Looking for floor - not found
-- Looking for floor in m
-- Looking for floor in m - found
-- Looking for gethostbyname_r
-- Looking for gethostbyname_r - found
-- Looking for bind
-- Looking for bind - found
-- Looking for crypt
-- Looking for crypt - not found
-- Looking for crypt in crypt
-- Looking for crypt in crypt - found
-- Looking for setsockopt
-- Looking for setsockopt - found
-- Looking for dlopen
-- Looking for dlopen - not found
-- Looking for dlopen in dl
-- Looking for dlopen in dl - found
-- Looking for sched_yield
-- Looking for sched_yield - found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for timer_create
-- Looking for timer_create - not found
-- Looking for timer_create in rt
-- Looking for timer_create in rt - found
-- Looking for include file alloca.h
-- Looking for include file alloca.h - found
-- Looking for include file arpa/inet.h
-- Looking for include file arpa/inet.h - found
-- Looking for include file dirent.h
-- Looking for include file dirent.h - found
-- Looking for include file dlfcn.h
-- Looking for include file dlfcn.h - found
-- Looking for include file execinfo.h
-- Looking for include file execinfo.h - found
-- Looking for include file fpu_control.h
-- Looking for include file fpu_control.h - found
-- Looking for include file grp.h
-- Looking for include file grp.h - found
-- Looking for include file ieeefp.h
-- Looking for include file ieeefp.h - not found
-- Looking for include file langinfo.h
-- Looking for include file langinfo.h - found
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for include file netinet/in.h
-- Looking for include file netinet/in.h - found
-- Looking for include file poll.h
-- Looking for include file poll.h - found
-- Looking for include file pwd.h
-- Looking for include file pwd.h - found
-- Looking for include file strings.h
-- Looking for include file strings.h - found
-- Looking for include file sys/cdefs.h
-- Looking for include file sys/cdefs.h - found
-- Looking for include file sys/ioctl.h
-- Looking for include file sys/ioctl.h - found
-- Looking for include file sys/mman.h
-- Looking for include file sys/mman.h - found
-- Looking for include file sys/resource.h
-- Looking for include file sys/resource.h - found
-- Looking for include file sys/select.h
-- Looking for include file sys/select.h - found
-- Looking for include file sys/socket.h
-- Looking for include file sys/socket.h - found
-- Looking for include files curses.h, term.h
-- Looking for include files curses.h, term.h - found
-- Looking for include file termios.h
-- Looking for include file termios.h - found
-- Looking for include file termio.h
-- Looking for include file termio.h - found
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for include file sys/wait.h
-- Looking for include file sys/wait.h - found
-- Looking for include file sys/param.h
-- Looking for include file sys/param.h - found
-- Looking for include file fnmatch.h
-- Looking for include file fnmatch.h - found
-- Looking for include file sys/un.h
-- Looking for include file sys/un.h - found
-- Looking for include file vis.h
-- Looking for include file vis.h - not found
-- Looking for include file sasl/sasl.h
-- Looking for include file sasl/sasl.h - found
-- Looking for include file sys/devpoll.h
-- Looking for include file sys/devpoll.h - not found
-- Looking for include file sys/epoll.h
-- Looking for include file sys/epoll.h - found
-- Looking for TAILQ_FOREACH
-- Looking for TAILQ_FOREACH - found
-- Looking for _aligned_malloc
-- Looking for _aligned_malloc - not found
-- Looking for backtrace
-- Looking for backtrace - found
-- Looking for printstack
-- Looking for printstack - not found
-- Looking for index
-- Looking for index - found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for cuserid
-- Looking for cuserid - found
-- Looking for directio
-- Looking for directio - not found
-- Looking for ftruncate
-- Looking for ftruncate - found
-- Looking for compress
-- Looking for compress - not found
-- Looking for crypt
-- Looking for crypt - found
-- Looking for dlopen
-- Looking for dlopen - found
-- Looking for fchmod
-- Looking for fchmod - found
-- Looking for fcntl
-- Looking for fcntl - found
-- Looking for fdatasync
-- Looking for fdatasync - found
-- Looking for fdatasync
-- Looking for fdatasync - found
-- Looking for fedisableexcept
-- Looking for fedisableexcept - found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for fsync
-- Looking for fsync - found
-- Looking for gethostbyaddr_r
-- Looking for gethostbyaddr_r - found
-- Looking for gethrtime
-- Looking for gethrtime - not found
-- Looking for getnameinfo
-- Looking for getnameinfo - found
-- Looking for getpass
-- Looking for getpass - found
-- Looking for getpassphrase
-- Looking for getpassphrase - not found
-- Looking for getpwnam
-- Looking for getpwnam - found
-- Looking for getpwuid
-- Looking for getpwuid - found
-- Looking for getrlimit
-- Looking for getrlimit - found
-- Looking for getrusage
-- Looking for getrusage - found
-- Looking for initgroups
-- Looking for initgroups - found
-- Looking for issetugid
-- Looking for issetugid - not found
-- Looking for getuid
-- Looking for getuid - found
-- Looking for geteuid
-- Looking for geteuid - found
-- Looking for getgid
-- Looking for getgid - found
-- Looking for getegid
-- Looking for getegid - found
-- Looking for lstat
-- Looking for lstat - found
-- Looking for madvise
-- Looking for madvise - found
-- Looking for malloc_info
-- Looking for malloc_info - found
-- Looking for memrchr
-- Looking for memrchr - found
-- Looking for mlock
-- Looking for mlock - found
-- Looking for mlockall
-- Looking for mlockall - found
-- Looking for mmap64
-- Looking for mmap64 - found
-- Looking for poll
-- Looking for poll - found
-- Looking for posix_fallocate
-- Looking for posix_fallocate - found
-- Looking for posix_memalign
-- Looking for posix_memalign - found
-- Looking for pread
-- Looking for pread - found
-- Looking for pthread_condattr_setclock
-- Looking for pthread_condattr_setclock - found
-- Looking for pthread_sigmask
-- Looking for pthread_sigmask - found
-- Looking for readdir_r
-- Looking for readdir_r - found
-- Looking for readlink
-- Looking for readlink - found
-- Looking for realpath
-- Looking for realpath - found
-- Looking for setfd
-- Looking for setfd - not found
-- Looking for sigaction
-- Looking for sigaction - found
-- Looking for sleep
-- Looking for sleep - found
-- Looking for stpcpy
-- Looking for stpcpy - found
-- Looking for stpncpy
-- Looking for stpncpy - found
-- Looking for strlcpy
-- Looking for strlcpy - not found
-- Looking for strnlen
-- Looking for strnlen - found
-- Looking for strlcat
-- Looking for strlcat - not found
-- Looking for strsignal
-- Looking for strsignal - found
-- Looking for fgetln
-- Looking for fgetln - not found
-- Looking for strsep
-- Looking for strsep - found
-- Looking for tell
-- Looking for tell - not found
-- Looking for vasprintf
-- Looking for vasprintf - found
-- Looking for memalign
-- Looking for memalign - found
-- Looking for nl_langinfo
-- Looking for nl_langinfo - found
-- Looking for ntohll
-- Looking for ntohll - not found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for epoll_create
-- Looking for epoll_create - found
-- Looking for inet_ntop
-- Looking for inet_ntop - found
-- Looking for kqueue
-- Looking for kqueue - not found
-- Looking for timeradd
-- Looking for timeradd - found
-- Looking for timerclear
-- Looking for timerclear - found
-- Looking for timercmp
-- Looking for timercmp - found
-- Looking for timerisset
-- Looking for timerisset - found
-- Looking for include file sys/time.h
-- Looking for include file sys/time.h - found
-- Looking for include file sys/times.h
-- Looking for include file sys/times.h - found
-- Looking for times
-- Looking for times - found
-- Looking for gettimeofday
-- Looking for gettimeofday - found
-- Looking for lrand48
-- Looking for lrand48 - found
-- Looking for TIOCGWINSZ
-- Looking for TIOCGWINSZ - found
-- Looking for FIONREAD
-- Looking for FIONREAD - found
-- Looking for FIONREAD
-- Looking for FIONREAD - not found
-- Looking for SIGEV_THREAD_ID
-- Looking for SIGEV_THREAD_ID - found
-- Looking for SIGEV_PORT
-- Looking for SIGEV_PORT - not found
-- Looking for log2
-- Looking for log2 - found
-- Performing Test HAVE_ISINF
-- Performing Test HAVE_ISINF - Success
-- Looking for timer_create
-- Looking for timer_create - found
-- Looking for timer_settime
-- Looking for timer_settime - found
-- Looking for kqueue
-- Looking for kqueue - not found
-- Looking for EVFILT_TIMER
-- Looking for EVFILT_TIMER - not found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Check size of char *
-- Check size of char * - done
-- Check size of long
-- Check size of long - done
-- Check size of short
-- Check size of short - done
-- Check size of int
-- Check size of int - done
-- Check size of long long
-- Check size of long long - done
-- Check size of off_t
-- Check size of off_t - done
-- Check size of time_t
-- Check size of time_t - done
-- Check size of uint
-- Check size of uint - done
-- Check size of ulong
-- Check size of ulong - done
-- Check size of u_int32_t
-- Check size of u_int32_t - done
-- Checking stack direction : -1
-- Looking for include files time.h, sys/time.h
-- Looking for include files time.h, sys/time.h - found
-- Looking for O_NONBLOCK
-- Looking for O_NONBLOCK - found
-- Performing Test HAVE_PAUSE_INSTRUCTION
-- Performing Test HAVE_PAUSE_INSTRUCTION - Success
-- Performing Test HAVE_BSS_START
-- Performing Test HAVE_BSS_START - Success
-- Performing Test HAVE_BUILTIN_UNREACHABLE
-- Performing Test HAVE_BUILTIN_UNREACHABLE - Success
-- Performing Test HAVE_BUILTIN_EXPECT
-- Performing Test HAVE_BUILTIN_EXPECT - Success
-- Performing Test HAVE_BUILTIN_STPCPY
-- Performing Test HAVE_BUILTIN_STPCPY - Success
-- Performing Test HAVE_GCC_ATOMIC_BUILTINS
-- Performing Test HAVE_GCC_ATOMIC_BUILTINS - Success
-- Looking for netinet/in6.h
-- Looking for netinet/in6.h - not found
-- Check size of struct sockaddr_in6
-- Check size of struct sockaddr_in6 - done
-- Check size of struct in6_addr
-- Check size of struct in6_addr - done
-- Performing Test HAVE_SOCKADDR_STORAGE_SS_FAMILY
-- Performing Test HAVE_SOCKADDR_STORAGE_SS_FAMILY - Success
-- Performing Test HAVE_SOCKADDR_IN_SIN_LEN
-- Performing Test HAVE_SOCKADDR_IN_SIN_LEN - Failed
-- Performing Test HAVE_SOCKADDR_IN6_SIN6_LEN
-- Performing Test HAVE_SOCKADDR_IN6_SIN6_LEN - Failed
-- Performing Test HAVE_IMPLICIT_DEPENDENT_NAME_TYPING
-- Performing Test HAVE_IMPLICIT_DEPENDENT_NAME_TYPING - Failed
-- Performing Test HAVE_VISIBILITY_HIDDEN
-- Performing Test HAVE_VISIBILITY_HIDDEN - Success
-- Performing Test HAVE_WVLA
-- Performing Test HAVE_WVLA - Success
-- Library mysqlclient depends on OSLIBS -lpthread;m;rt;dl
-- CMAKE_BUILD_TYPE: RelWithDebInfo
-- COMPILE_DEFINITIONS: _GNU_SOURCE;_FILE_OFFSET_BITS=64;HAVE_CONFIG_H
-- CMAKE_C_FLAGS:  -Wall -Wextra -Wformat-security -Wvla -Wwrite-strings -Wdeclaration-after-statement
-- CMAKE_CXX_FLAGS:  -Wall -Wextra -Wformat-security -Wvla -Woverloaded-virtual -Wno-unused-parameter
-- CMAKE_C_FLAGS_RELWITHDEBINFO: -O3 -g -fabi-version=2 -fno-omit-frame-pointer -fno-strict-aliasing -DDBUG_OFF
-- CMAKE_CXX_FLAGS_RELWITHDEBINFO: -O3 -g -fabi-version=2 -fno-omit-frame-pointer -fno-strict-aliasing -DDBUG_OFF
-- Configuring done
-- Generating done
-- Build files have been written to: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native
make -C '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native' -j '4' VERBOSE=1
make[2]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -H/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src -B/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native --check-build-system CMakeFiles/Makefile.cmake 0
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_start /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/progress.marks
make -f CMakeFiles/Makefile2 all
make[3]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
make -f CMakeFiles/INFO_BIN.dir/build.make CMakeFiles/INFO_BIN.dir/depend
make -f CMakeFiles/INFO_SRC.dir/build.make CMakeFiles/INFO_SRC.dir/depend
make -f CMakeFiles/abi_check.dir/build.make CMakeFiles/abi_check.dir/depend
make -f zlib/CMakeFiles/zlib.dir/build.make zlib/CMakeFiles/zlib.dir/depend
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native && /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_depends "Unix Makefiles" /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/abi_check.dir/DependInfo.cmake --color=
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native && /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_depends "Unix Makefiles" /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/zlib /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/zlib /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/zlib/CMakeFiles/zlib.dir/DependInfo.cmake --color=
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native && /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_depends "Unix Makefiles" /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_BIN.dir/DependInfo.cmake --color=
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native && /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_depends "Unix Makefiles" /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_SRC.dir/DependInfo.cmake --color=
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/zlib/CMakeFiles/zlib.dir/DependInfo.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/zlib/CMakeFiles/zlib.dir/depend.internal".
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/zlib/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/zlib/CMakeFiles/zlib.dir/depend.internal".
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/abi_check.dir/DependInfo.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/abi_check.dir/depend.internal".
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/abi_check.dir/depend.internal".
Scanning dependencies of target abi_check
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
make -f CMakeFiles/abi_check.dir/build.make CMakeFiles/abi_check.dir/build
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_BIN.dir/DependInfo.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_BIN.dir/depend.internal".
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_BIN.dir/depend.internal".
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_SRC.dir/DependInfo.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_SRC.dir/depend.internal".
Dependee "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles/INFO_SRC.dir/depend.internal".
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -DCOMPILER=/usr/bin/cc -DSOURCE_DIR=/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src -DBINARY_DIR=/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native "-DABI_HEADERS=/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/include/mysql.h;/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/include/mysql/psi/psi_abi_v0.h;/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/include/mysql/psi/psi_abi_v1.h;/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/include/mysql/psi/psi_abi_v2.h;/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/include/mysql/client_plugin.h" -P /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/cmake/do_abi_check.cmake
Scanning dependencies of target INFO_SRC
Scanning dependencies of target INFO_BIN
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
make -f CMakeFiles/INFO_BIN.dir/build.make CMakeFiles/INFO_BIN.dir/build
make -f CMakeFiles/INFO_SRC.dir/build.make CMakeFiles/INFO_SRC.dir/build
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -P /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/cmake/info_bin.cmake
Scanning dependencies of target zlib
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -P /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/cmake/info_src.cmake
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
make -f zlib/CMakeFiles/zlib.dir/build.make zlib/CMakeFiles/zlib.dir/build
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles 95
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles 
[  1%] [  1%] Building C object zlib/CMakeFiles/zlib.dir/adler32.c.o

[...]

[100%] Generating libmysqlclient_r.a
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/libmysql && /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E remove -f /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/libmysql/libmysqlclient_r.a
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/libmysql && /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E create_symlink libmysqlclient.a libmysqlclient_r.a
Scanning dependencies of target mysql_client_test
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
make -f tests/CMakeFiles/mysql_client_test.dir/build.make tests/CMakeFiles/mysql_client_test.dir/build
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles  76
make[4]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles 
[100%] [100%] Built target symlink_libmysqlclient_r.a
Building C object tests/CMakeFiles/mysql_client_test.dir/mysql_client_test.c.o
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/tests && /usr/bin/cc  -DHAVE_CONFIG_H -DMYSQL_CLIENT -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE -Wall -Wextra -Wformat-security -Wvla -Wwrite-strings -Wdeclaration-after-statement -O3 -g -fabi-version=2 -fno-omit-frame-pointer -fno-strict-aliasing -DDBUG_OFF -I/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/include -I/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/extra/rapidjson/include -I/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/include    -o CMakeFiles/mysql_client_test.dir/mysql_client_test.c.o   -c /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src/tests/mysql_client_test.c
Linking CXX executable mysql_client_test
cd /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/tests && /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_link_script CMakeFiles/mysql_client_test.dir/link.txt --verbose=1
/usr/bin/c++    -Wall -Wextra -Wformat-security -Wvla -Woverloaded-virtual -Wno-unused-parameter -O3 -g -fabi-version=2 -fno-omit-frame-pointer -fno-strict-aliasing -DDBUG_OFF    CMakeFiles/mysql_client_test.dir/mysql_client_test.c.o  -o mysql_client_test  -lpthread ../libmysql/libmysqlclient.a -lpthread -lm -lrt -ldl 
make[4]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles 
[100%] Built target mysql_client_test
make[3]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_start /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/CMakeFiles 0
make[2]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native'
# cross-compilation
mkdir '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build'
cd '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build' && cmake -DCMAKE_INSTALL_PREFIX=/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/i686-w64-mingw32.static -DCMAKE_TOOLCHAIN_FILE='/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/i686-w64-mingw32.static/share/cmake/mxe-conf.cmake' -DIMPORT_COMP_ERR='/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.native/ImportCompErr.cmake' -DHAVE_GCC_ATOMIC_BUILTINS=1 -DDISABLE_SHARED=ON -DENABLE_DTRACE=OFF -DWITH_ZLIB=system '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src'
-- Running cmake version 3.0.2
-- The C compiler identification is GNU 4.9.3
-- The CXX compiler identification is GNU 4.9.3
-- Check for working C compiler: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-gcc
-- Check for working C compiler: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-g++
-- Check for working CXX compiler: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void *
-- Check size of void * - done
-- SIZEOF_VOIDP 4
-- LibMySQL 6.1.6
-- Built from MySQL 5.7.6-m16 sources
-- Packaging as: mysql-connector-c-6.1.6-win32
-- Installing to: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/i686-w64-mingw32.static
Dynamic plugins are disabled.
-- Found Threads: TRUE  
-- Looking for getnameinfo
-- Looking for getnameinfo - not found
-- Looking for inet_ntop
-- Looking for inet_ntop - not found
-- Looking for log2
-- Looking for log2 - found
-- Performing Test HAVE_ISINF
-- Performing Test HAVE_ISINF - Success
-- Check size of struct sockaddr_in6
-- Check size of struct sockaddr_in6 - done
-- Check size of struct in6_addr
-- Check size of struct in6_addr - done
-- Performing Test HAVE_IMPLICIT_DEPENDENT_NAME_TYPING
-- Performing Test HAVE_IMPLICIT_DEPENDENT_NAME_TYPING - Failed
-- Looking for crc32
-- Looking for crc32 - found
-- Looking for compressBound
-- Looking for compressBound - found
-- Looking for deflateBound
-- Looking for deflateBound - found
-- Performing Test HAVE_WVLA
-- Performing Test HAVE_WVLA - Success
-- Library mysqlclient depends on OSLIBS ws2_32;/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/i686-w64-mingw32.static/lib/libz.a;secur32
RegularExpression::compile(): Nested *?+.
RegularExpression::compile(): Error in compile.
CMake Error at scripts/CMakeLists.txt:62 (IF):
  if given arguments:

    "NOT" "lib" "STREQUAL" "general" "AND" "NOT" "CLIENT_LIBS" "MATCHES" "-l/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/i686-w64-mingw32.static/lib/libz.a "

  Regular expression
  "-l/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/i686-w64-mingw32.static/lib/libz.a
  " cannot compile
Call Stack (most recent call first):
  scripts/CMakeLists.txt:81 (EXTRACT_LINK_LIBRARIES)


-- Configuring incomplete, errors occurred!
See also "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeOutput.log".
See also "/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeError.log".
Makefile:505: recipe for target 'build-only-libmysqlclient_i686-w64-mingw32.static' failed
make[1]: *** [build-only-libmysqlclient_i686-w64-mingw32.static] Error 1
make[1]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe'

real	0m54.336s
user	1m27.756s
sys	0m14.613s
```

CMakeError.log:
```
Determining if the function getnameinfo exists failed with the following output:
Change Dir: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/gmake" "cmTryCompileExec2540705121/fast"
gmake[2]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
/usr/bin/gmake -f CMakeFiles/cmTryCompileExec2540705121.dir/build.make CMakeFiles/cmTryCompileExec2540705121.dir/build
gmake[3]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/CMakeFiles 1
Building C object CMakeFiles/cmTryCompileExec2540705121.dir/CheckFunctionExists.c.obj
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-gcc   -DCHECK_FUNCTION_EXISTS=getnameinfo  -D_WIN32_WINNT=0x0601 -O3 -DNDEBUG   -o CMakeFiles/cmTryCompileExec2540705121.dir/CheckFunctionExists.c.obj   -c /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/share/cmake-3.0/Modules/CheckFunctionExists.c
Linking C executable cmTryCompileExec2540705121.exe
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_link_script CMakeFiles/cmTryCompileExec2540705121.dir/link.txt --verbose=1
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E remove -f CMakeFiles/cmTryCompileExec2540705121.dir/objects.a
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-ar cr CMakeFiles/cmTryCompileExec2540705121.dir/objects.a @CMakeFiles/cmTryCompileExec2540705121.dir/objects1.rsp
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-gcc   -DCHECK_FUNCTION_EXISTS=getnameinfo  -D_WIN32_WINNT=0x0601 -O3 -DNDEBUG    -Wl,--whole-archive CMakeFiles/cmTryCompileExec2540705121.dir/objects.a -Wl,--no-whole-archive  -o cmTryCompileExec2540705121.exe -Wl,--out-implib,libcmTryCompileExec2540705121.dll.a -Wl,--major-image-version,0,--minor-image-version,0  -lws2_32 -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 
CMakeFiles/cmTryCompileExec2540705121.dir/objects.a(CheckFunctionExists.c.obj):CheckFunctionExists.c:(.text.startup+0xc): undefined reference to `getnameinfo'
collect2: error: ld returned 1 exit status
CMakeFiles/cmTryCompileExec2540705121.dir/build.make:89: recipe for target 'cmTryCompileExec2540705121.exe' failed
gmake[3]: *** [cmTryCompileExec2540705121.exe] Error 1
gmake[3]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
Makefile:118: recipe for target 'cmTryCompileExec2540705121/fast' failed
gmake[2]: *** [cmTryCompileExec2540705121/fast] Error 2
gmake[2]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'


Determining if the function inet_ntop exists failed with the following output:
Change Dir: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/gmake" "cmTryCompileExec1725997306/fast"
gmake[2]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
/usr/bin/gmake -f CMakeFiles/cmTryCompileExec1725997306.dir/build.make CMakeFiles/cmTryCompileExec1725997306.dir/build
gmake[3]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/CMakeFiles 1
Building C object CMakeFiles/cmTryCompileExec1725997306.dir/CheckFunctionExists.c.obj
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-gcc   -DCHECK_FUNCTION_EXISTS=inet_ntop  -D_WIN32_WINNT=0x0601 -O3 -DNDEBUG   -o CMakeFiles/cmTryCompileExec1725997306.dir/CheckFunctionExists.c.obj   -c /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/share/cmake-3.0/Modules/CheckFunctionExists.c
Linking C executable cmTryCompileExec1725997306.exe
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_link_script CMakeFiles/cmTryCompileExec1725997306.dir/link.txt --verbose=1
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E remove -f CMakeFiles/cmTryCompileExec1725997306.dir/objects.a
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-ar cr CMakeFiles/cmTryCompileExec1725997306.dir/objects.a @CMakeFiles/cmTryCompileExec1725997306.dir/objects1.rsp
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-gcc   -DCHECK_FUNCTION_EXISTS=inet_ntop  -D_WIN32_WINNT=0x0601 -O3 -DNDEBUG    -Wl,--whole-archive CMakeFiles/cmTryCompileExec1725997306.dir/objects.a -Wl,--no-whole-archive  -o cmTryCompileExec1725997306.exe -Wl,--out-implib,libcmTryCompileExec1725997306.dll.a -Wl,--major-image-version,0,--minor-image-version,0  -lws2_32 -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 
CMakeFiles/cmTryCompileExec1725997306.dir/objects.a(CheckFunctionExists.c.obj):CheckFunctionExists.c:(.text.startup+0xc): undefined reference to `inet_ntop'
collect2: error: ld returned 1 exit status
CMakeFiles/cmTryCompileExec1725997306.dir/build.make:89: recipe for target 'cmTryCompileExec1725997306.exe' failed
gmake[3]: *** [cmTryCompileExec1725997306.exe] Error 1
gmake[3]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
Makefile:118: recipe for target 'cmTryCompileExec1725997306/fast' failed
gmake[2]: *** [cmTryCompileExec1725997306/fast] Error 2
gmake[2]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'


Performing C++ SOURCE FILE Test HAVE_IMPLICIT_DEPENDENT_NAME_TYPING failed with the following output:
Change Dir: /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/gmake" "cmTryCompileExec2125492140/fast"
gmake[2]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
/usr/bin/gmake -f CMakeFiles/cmTryCompileExec2125492140.dir/build.make CMakeFiles/cmTryCompileExec2125492140.dir/build
gmake[3]: Entering directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/x86_64-unknown-linux-gnu/bin/cmake -E cmake_progress_report /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/CMakeFiles 1
Building CXX object CMakeFiles/cmTryCompileExec2125492140.dir/src.cxx.obj
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/usr/bin/i686-w64-mingw32.static-g++   -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGE_FILES -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -DHAVE_IMPLICIT_DEPENDENT_NAME_TYPING  -D_WIN32_WINNT=0x0601 -O3 -DNDEBUG   -o CMakeFiles/cmTryCompileExec2125492140.dir/src.cxx.obj -c /home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx: In member function 'void ct2<T>::func()':
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx:15:5: error: need 'typename' before 'std::vector<T>::iterator' because 'std::vector<T>' is a dependent scope
     std::vector<T>::iterator itr = vec.begin();
     ^
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx:15:30: error: expected ';' before 'itr'
     std::vector<T>::iterator itr = vec.begin();
                              ^
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx: In instantiation of 'void ct2<T>::func() [with T = double]':
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx:21:13:   required from here
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx:15:5: error: dependent-name 'std::vector<T>::iterator' is parsed as a non-type, but instantiation yields a type
     std::vector<T>::iterator itr = vec.begin();
     ^
/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp/src.cxx:15:5: note: say 'typename std::vector<T>::iterator' if a type is meant
CMakeFiles/cmTryCompileExec2125492140.dir/build.make:57: recipe for target 'CMakeFiles/cmTryCompileExec2125492140.dir/src.cxx.obj' failed
gmake[3]: *** [CMakeFiles/cmTryCompileExec2125492140.dir/src.cxx.obj] Error 1
gmake[3]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'
Makefile:118: recipe for target 'cmTryCompileExec2125492140/fast' failed
gmake[2]: *** [cmTryCompileExec2125492140/fast] Error 2
gmake[2]: Leaving directory '/home/lukas/Programmieren/C++/Qt/Chat/win32/mxe/tmp-libmysqlclient-i686-w64-mingw32.static/mysql-connector-c-6.1.6-src.build/CMakeFiles/CMakeTmp'

Source file was:

  #include <vector>
  template<typename T>
  class ct2
  {
  public:
    typedef T type;
    void func();
  };

  template<typename T>
  void ct2<T>::func()
  {
    std::vector<T> vec;
    std::vector<T>::iterator itr = vec.begin();
  }

  int main(int argc, char **argv)
  {
    ct2<double> o2;
    o2.func();
    return 0;
  }
```

Grrr, it doesn't like a `++` in the path, I can replicate the same here:
```
-- Performing Test HAVE_WVLA
-- Performing Test HAVE_WVLA - Success
-- Library mysqlclient depends on OSLIBS ws2_32;/Users/tonyt/dev/mxe-++/usr/i686-w64-mingw32.static/lib/libz.a;secur32
RegularExpression::compile(): Nested *?+.
RegularExpression::compile(): Error in compile.
CMake Error at scripts/CMakeLists.txt:62 (IF):
  if given arguments:

    "NOT" "lib" "STREQUAL" "general" "AND" "NOT" "CLIENT_LIBS" "MATCHES" "-l/Users/tonyt/dev/mxe-++/usr/i686-w64-mingw32.static/lib/libz.a "

  Regular expression
  "-l/Users/tonyt/dev/mxe-++/usr/i686-w64-mingw32.static/lib/libz.a " cannot
  compile
Call Stack (most recent call first):
  scripts/CMakeLists.txt:81 (EXTRACT_LINK_LIBRARIES)


-- Configuring incomplete, errors occurred!
```

--- a/src/libmysqlclient-1-fixes.patch
+++ b/src/libmysqlclient-1-fixes.patch
@@ -348,3 +348,26 @@ index 1111111..2222222 100644
    double t= d - (double) 0x8000000000000000ULL;
  
 
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Boris Nagaev <bnagaev@gmail.com>
+Date: Wed, 16 Dec 2015 17:39:28 +0100
+Subject: [PATCH] fix CMake error if dir contains special regex char
+
+See https://github.com/mxe/mxe/issues/1030
+
+diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
+index 1111111..2222222 100644
+--- a/scripts/CMakeLists.txt
++++ b/scripts/CMakeLists.txt
+@@ -59,7 +59,10 @@ MACRO(EXTRACT_LINK_LIBRARIES target var)
+     FOREACH(lib ${${target}_LIB_DEPENDS})
+       # Filter out "general", it is not a library, just CMake hint
+       # Also, remove duplicates
+-      IF(NOT lib STREQUAL "general" AND NOT ${var}  MATCHES "-l${lib} ")
++      # To check if "${var}" contains "-l${lib} ", use replace
++      # can't use MATCHES because ${lib} may contain special chars.
++      STRING(REPLACE "-l${lib} " "" var_repl ${var})
++      IF(NOT lib STREQUAL "general" AND ${var} STREQUAL ${var_repl})
+         IF (lib MATCHES "^\\-l")
+           SET(${var} "${${var}} ${lib} ") 
+         ELSEIF(lib MATCHES "^/")

--- a/src/libmysqlclient-1-fixes.patch
+++ b/src/libmysqlclient-1-fixes.patch
@@ -3,15 +3,15 @@ See index.html for further information.
 
 Contains ad hoc patches for cross building.
 
-From 93f3a29abe0397e5a6681c07c3223372b5451bb0 Mon Sep 17 00:00:00 2001
-From: MXE
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "fix@me" <fix@me>
 Date: Sun, 6 Sep 2015 22:34:47 +0100
-Subject: [PATCH 1/7] use comp_err tool built in native build
+Subject: [PATCH] use comp_err tool built in native build
 
 See http://www.vtk.org/Wiki/CMake_Cross_Compiling#Using_executables_in_the_build_created_during_the_build
 
 diff --git a/extra/CMakeLists.txt b/extra/CMakeLists.txt
-index ac28c62..b2cbb75 100644
+index 1111111..2222222 100644
 --- a/extra/CMakeLists.txt
 +++ b/extra/CMakeLists.txt
 @@ -20,13 +20,29 @@ ${ZLIB_INCLUDE_DIR})
@@ -44,19 +44,16 @@ index ac28c62..b2cbb75 100644
  ADD_CUSTOM_COMMAND(OUTPUT ${PROJECT_BINARY_DIR}/include/mysqld_error.h 
                     ${PROJECT_BINARY_DIR}/sql/share/english/errmsg.sys
                     COMMAND comp_err
--- 
-2.3.8 (Apple Git-58)
 
-
-From e45e5c21360b12cb9f271a29bd1d4d183ed55060 Mon Sep 17 00:00:00 2001
-From: MXE
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "fix@me" <fix@me>
 Date: Sun, 6 Sep 2015 22:53:33 +0100
-Subject: [PATCH 2/7] fix extra qualification 'Handshake::'
+Subject: [PATCH] fix extra qualification 'Handshake::'
 
 Compilation error.
 
 diff --git a/libmysql/authentication_win/handshake.h b/libmysql/authentication_win/handshake.h
-index 14b1386..395af36 100644
+index 1111111..2222222 100644
 --- a/libmysql/authentication_win/handshake.h
 +++ b/libmysql/authentication_win/handshake.h
 @@ -100,7 +100,7 @@ public:
@@ -68,18 +65,15 @@ index 14b1386..395af36 100644
  
    bool virtual is_complete() const
    {
--- 
-2.3.8 (Apple Git-58)
 
-
-From a2deca6cb6f87bb926d12149595c84f29ae00d0e Mon Sep 17 00:00:00 2001
-From: MXE
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "fix@me" <fix@me>
 Date: Mon, 7 Sep 2015 22:59:18 +0100
-Subject: [PATCH 3/7] fix case in headers and libs (should be lowercase)
+Subject: [PATCH] fix case in headers and libs (should be lowercase)
 
 
 diff --git a/extra/yassl/src/yassl_int.cpp b/extra/yassl/src/yassl_int.cpp
-index 2993a1a..04b1fda 100644
+index 1111111..2222222 100644
 --- a/extra/yassl/src/yassl_int.cpp
 +++ b/extra/yassl/src/yassl_int.cpp
 @@ -20,7 +20,7 @@
@@ -92,7 +86,7 @@ index 2993a1a..04b1fda 100644
  #include <pthread.h>
  #endif
 diff --git a/include/mysql/psi/mysql_socket.h b/include/mysql/psi/mysql_socket.h
-index 41a7cb4..5f6d91d 100644
+index 1111111..2222222 100644
 --- a/include/mysql/psi/mysql_socket.h
 +++ b/include/mysql/psi/mysql_socket.h
 @@ -29,7 +29,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
@@ -105,7 +99,7 @@ index 41a7cb4..5f6d91d 100644
  #else
    #include <netinet/in.h>
 diff --git a/libmysql/authentication_win/CMakeLists.txt b/libmysql/authentication_win/CMakeLists.txt
-index f2979fb..83e5441 100644
+index 1111111..2222222 100644
 --- a/libmysql/authentication_win/CMakeLists.txt
 +++ b/libmysql/authentication_win/CMakeLists.txt
 @@ -26,7 +26,7 @@ SET(HEADERS common.h handshake.h)
@@ -118,7 +112,7 @@ index f2979fb..83e5441 100644
  # In IDE, group headers in a separate folder.
  
 diff --git a/libmysql/authentication_win/plugin_client.cc b/libmysql/authentication_win/plugin_client.cc
-index d25aae8..ecfee79 100644
+index 1111111..2222222 100644
 --- a/libmysql/authentication_win/plugin_client.cc
 +++ b/libmysql/authentication_win/plugin_client.cc
 @@ -26,7 +26,7 @@
@@ -130,20 +124,17 @@ index d25aae8..ecfee79 100644
  #endif
  
  static int win_auth_client_plugin_init(char*, size_t, int, va_list)
--- 
-2.3.8 (Apple Git-58)
 
-
-From 7517a9a4ce9b123470ff8cc040acfa1777acc832 Mon Sep 17 00:00:00 2001
-From: MXE
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "fix@me" <fix@me>
 Date: Mon, 7 Sep 2015 23:01:35 +0100
-Subject: [PATCH 4/7] define missing types
+Subject: [PATCH] define missing types
 
 We do not use types CERT_NAME_BLOB and CRYPT_HASH_BLOB,
 though they are used in mprapi.h. And are not defined!
 
 diff --git a/mysys/my_gethwaddr.c b/mysys/my_gethwaddr.c
-index 780c765..e2cd321 100644
+index 1111111..2222222 100644
 --- a/mysys/my_gethwaddr.c
 +++ b/mysys/my_gethwaddr.c
 @@ -137,6 +137,9 @@ my_bool my_gethwaddr(uchar *to)
@@ -156,14 +147,11 @@ index 780c765..e2cd321 100644
  #include <iphlpapi.h>
  
  /*
--- 
-2.3.8 (Apple Git-58)
 
-
-From 12358baf55265ff6d2764771199645e811863853 Mon Sep 17 00:00:00 2001
-From: MXE
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "fix@me" <fix@me>
 Date: Mon, 7 Sep 2015 23:04:23 +0100
-Subject: [PATCH 5/7] macro NATIVE_WIN32 for stuff we don't have here
+Subject: [PATCH] macro NATIVE_WIN32 for stuff we don't have here
 
 Macro NATIVE_WIN32 is undefined in MXE. It is used instead of _WIN32
 in #ifdef's where working variant is Unix and not Win32:
@@ -173,7 +161,7 @@ in #ifdef's where working variant is Unix and not Win32:
  * MXE has Unix-like stacktrace reader
 
 diff --git a/include/m_string.h b/include/m_string.h
-index e1576af..5417192 100644
+index 1111111..2222222 100644
 --- a/include/m_string.h
 +++ b/include/m_string.h
 @@ -127,7 +127,7 @@ static inline ulonglong my_strtoull(const char *nptr, char **endptr, int base)
@@ -186,7 +174,7 @@ index e1576af..5417192 100644
  #else
    return strtok_r(str, delim, saveptr);
 diff --git a/include/my_global.h b/include/my_global.h
-index acadb44..8c3c3cd 100644
+index 1111111..2222222 100644
 --- a/include/my_global.h
 +++ b/include/my_global.h
 @@ -205,7 +205,7 @@ C_MODE_START
@@ -236,7 +224,7 @@ index acadb44..8c3c3cd 100644
      ts1->tv_nsec - ts2->tv_nsec;
  #else
 diff --git a/include/thr_cond.h b/include/thr_cond.h
-index 480e936..03fbc24 100644
+index 1111111..2222222 100644
 --- a/include/thr_cond.h
 +++ b/include/thr_cond.h
 @@ -39,7 +39,7 @@ typedef CONDITION_VARIABLE native_cond_t;
@@ -258,7 +246,7 @@ index 480e936..03fbc24 100644
      return ETIMEDOUT;
    return 0;
 diff --git a/mysys/stacktrace.c b/mysys/stacktrace.c
-index 226d469..db94096 100644
+index 1111111..2222222 100644
 --- a/mysys/stacktrace.c
 +++ b/mysys/stacktrace.c
 @@ -15,7 +15,7 @@
@@ -270,18 +258,15 @@ index 226d469..db94096 100644
  #include "my_pthread.h"
  #include "m_string.h"
  #include <signal.h>
--- 
-2.3.8 (Apple Git-58)
 
-
-From 4f6b41efba4a862d34de3d03980d283d60c57c28 Mon Sep 17 00:00:00 2001
-From: MXE
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "fix@me" <fix@me>
 Date: Thu, 8 Oct 2015 13:30:53 +1100
-Subject: [PATCH 6/7] fix shared lib names
+Subject: [PATCH] fix shared lib names
 
 
 diff --git a/libmysql/CMakeLists.txt b/libmysql/CMakeLists.txt
-index 17c8c03..9e2980b 100644
+index 1111111..2222222 100644
 --- a/libmysql/CMakeLists.txt
 +++ b/libmysql/CMakeLists.txt
 @@ -216,7 +216,7 @@ MACRO(GET_TARGET_NAME target out_name)
@@ -311,18 +296,15 @@ index 17c8c03..9e2980b 100644
      # libtool compatability
      IF(CMAKE_SYSTEM_NAME MATCHES "FreeBSD" OR APPLE)
        SET(OS_SHARED_LIB_VERSION "${SHARED_LIB_MAJOR_VERSION}")
--- 
-2.3.8 (Apple Git-58)
 
-
-From d57fefef1a1eb15d14dc64d4c59267746f02369d Mon Sep 17 00:00:00 2001
-From: MXE
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: "fix@me" <fix@me>
 Date: Thu, 8 Oct 2015 19:36:35 +1100
-Subject: [PATCH 7/7] allow install of mysql_config
+Subject: [PATCH] allow install of mysql_config
 
 
 diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
-index 69d19b8..738563b 100644
+index 1111111..2222222 100644
 --- a/scripts/CMakeLists.txt
 +++ b/scripts/CMakeLists.txt
 @@ -92,7 +92,7 @@ ENDIF()
@@ -334,11 +316,8 @@ index 69d19b8..738563b 100644
    # On Windows, some .sh and some .pl.in files are configured 
    # The resulting files will have .pl extension (those are perl scripts)
  
--- 
-2.3.8 (Apple Git-58)
 
-
-From eb38e04f41ac341b162cfa0e97aa3ee1ce783777 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Boris Nagaev <bnagaev@gmail.com>
 Date: Wed, 11 Nov 2015 07:44:44 +0000
 Subject: [PATCH] add 'static' to some inline functions
@@ -347,7 +326,7 @@ Fix https://gist.github.com/starius/71d5f276d62e58f36d2f
 See also http://stackoverflow.com/a/8502046
 
 diff --git a/include/my_global.h b/include/my_global.h
-index 8c3c3cd..3cb24f6 100644
+index 1111111..2222222 100644
 --- a/include/my_global.h
 +++ b/include/my_global.h
 @@ -340,7 +340,7 @@ typedef socket_len_t SOCKET_SIZE_TYPE; /* Used by NDB */
@@ -368,6 +347,4 @@ index 8c3c3cd..3cb24f6 100644
  {
    double t= d - (double) 0x8000000000000000ULL;
  
--- 
-2.1.4
 

